libraries/tools/idl2k/src/main/kotlin/build.kt

```diff
@@ -18,16 +19,17 @@ class BuildWebIdl(val mdnCacheFile: File, val srcDir: File) {
             System.exit(1)
         }
 
-        return srcDir.walkTopDown().filter { it.isDirectory || it.extension == "idl" }.asSequence().filter { it.isFile }.toList().sortedBy { it.absolutePath }.fold(Repository(emptyMap(), emptyMap(), emptyMap(), emptyMap())) { acc, e ->
+        return srcDir.walkTopDown().filter { it.isDirectory || it.extension == "idl" }.asSequence().filter { it.isFile }.toList()
+            .sortedBy { it.absolutePath }.fold(Repository(emptyMap(), emptyMap(), emptyMap(), emptyMap())) { acc, e ->
             System.err.flush()
             System.err.println("Parsing ${e.absolutePath}")
-            val fileRepository = parseIDL(ANTLRFileStream(e.absolutePath, "UTF-8"))
+            val fileRepository = parseIDL(CharStreams.fromFileName(e.absolutePath, StandardCharsets.UTF_8))
```

Comment:
```suggestion
            val fileRepository = parseIDL(CharStreams.fromFileName(e.absolutePath, Charsets.UTF_8))
```
`Charsets` object available in `kotlin.text`